tests/test_templatestags.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+from django.test import TestCase
+from django import template
+from request.templatetags.request_admin import trunc, pie_chart
+from request.templatetags.request_tag import ActiveUserNode, active_users
+
+
+class RequestAdminTruncTest(TestCase):
+    def test_not_truncate(self):
+        string = "A" * 10
+        new_string = trunc(string, 20)
+        self.assertEqual(len(new_string), 10)
+
+    def test_truncate(self):
+        string = "A" * 10
+        new_string = trunc(string, 5)
+        self.assertLess(len(new_string), 6)
+
+
+class RequestAdminPieChart(TestCase):
+    def test_pie_chart(self):
+        inventory = ['apple', 'lemon', 'apple', 'orange', 'lemon', 'lemon']
+        pie_chart(inventory)
```

Comment:
The `pie_chart` template tag will generate a URL for the chart. We could assert this URL is correctly generated.

```python
url = pie_chart(inventory)
self.assertEqual(url, 'http://chart.google....')
```